common/test/acceptance/pages/lms/video/video.py

```diff
@@ -356,9 +356,9 @@ def click_player_button(self, button):
 
         self.q(css=button_selector).first.click()
 
-        button_states = {'play': 'playing', 'pause': 'pause'}
-        if button in button_states:
-            self.wait_for_state(button_states[button])
+        # button_states = {'play': 'playing', 'pause': 'pause'}
+        # if button in button_states:
+        #     self.wait_for_state(button_states[button])
```

Comment:
Removed unnecessary statement that leads to flakiness.
Video player's state can be changed many times per second + `try_interval` + CPU load, network latency = flaky test.